_spinnaker/armory_halyard.md

```diff
@@ -49,9 +49,75 @@ hal armory [subcommands]
 ```
 
 #### Subcommands
+ * `diagnostics`: configure diagnostics reporting
  * `dinghy`: configure Dinghy pipelines as code
  * `init`: Runs Armory installer
 
+---
+## hal armory diagnostics
+
+configure diagnostics reporting
+
+#### Usage
+```
+hal armory diagnostics [parameters] [subcommands]
+```
+
+#### Parameters
+ * `--deployment`: If supplied, use this Halyard deployment. This will _not_ create a new deployment.
+ * `--no-validate`: (*Default*: `false`) Skip validation.
+
+#### Subcommands
+ * `disable`: Set diagnostics as disabled
+ * `edit`: Edit diagnostics settings
+ * `enable`: Set diagnostics as enabled
+
+---
+## hal armory diagnostics disable
+
+Set diagnostics as disabled
+
+#### Usage
+```
+hal armory diagnostics disable [parameters]
+```
+
+#### Parameters
+ * `--deployment`: If supplied, use this Halyard deployment. This will _not_ create a new deployment.
+ * `--no-validate`: (*Default*: `false`) Skip validation.
+
+
+---
+## hal armory diagnostics edit
+
+Edit diagnostics settings
+
+#### Usage
+```
+hal armory diagnostics edit [parameters]
+```
+
+#### Parameters
+ * `--deployment`: If supplied, use this Halyard deployment. This will _not_ create a new deployment.
+ * `--no-validate`: (*Default*: `false`) Skip validation.
+ * `--uuid`: (*Required*) UUID of the Armory installation
```

Comment:
Will the customer know what to put into this required field?

Reply:
I don't think so. We use it in armory-install. I think this is more for internal use, but it is exposed anyway